src/Env/Config/Variable/MakeTarget.php

```diff
@@ -35,22 +35,17 @@ public function __construct($name, array $commands)
      */
     public function getReplaces()
     {
-        $replaces = '';
-
-        for ($i = 0; $i < count($this->commands); ++$i) {
-            $command = $this->commands[$i];
-            $replaces .= 0 < $i ? "\n\t$command" : $command;
-        }
-
         return [
-            sprintf('{{ %s_tasks }}', $this->name) => $replaces,
+            sprintf('{{ %s_tasks }}', $this->name) => array_reduce($this->commands, function ($previous, $command) {
```

Comment:
- `array_reduce`: https://3v4l.org/BKAA2/perf
- `for $i`: https://3v4l.org/M1QmV/perf

<img width="961" alt="screenshot 2016-10-05 a 23 29 01" src="https://cloud.githubusercontent.com/assets/2211145/19132406/8779eafc-8b53-11e6-96c1-659851ef57eb.PNG">